ui/oopsyraidsy/data/04-sb/trial/suzaku-ex.ts

```diff
@@ -0,0 +1,78 @@
+import NetRegexes from '../../../../../resources/netregexes';
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+import { playerDamageFields } from '../../../oopsy_common';
+
+// TODO: Rekindle (32E0) hitting multiple adds, any other players,
+// or hitting adds before they've been killed once is a mistake.
+// However, there is a point where it should hit one add in the
+// beginning. This also happens later in the fight too, where it
+// should not hit anybody else.
```

Comment:
As a stop-gap, maybe set a phase-change trigger and at least warn of shared Rekindle markers during the second half? Or if we want to handle it really properly, check the EntityID for shares and see whether both are players?

Reply:
Unfortunately, I think it's really hard to see if "both" are shares, because if all 4 people hit themselves and a bird and 1 extra person gets hit by it, then they're all `0x16`, and how do you know which one of those 9 lines is the failure?

I'm kind of inclined to just leave this as an unhandled TODO.

Reply:
Mmph, I see what you mean. It would end up being "check sources on each instance of damage, and for each source, at most one target can be a player". Not worth it. I do think it's worth handling post-intermission though?